### PR TITLE
Add ability for window to remember position and size

### PR DESCRIPTION
**Pull request by mleo2003**
Based on #59 , but updated to work with latest code.

I'm not happy with one part of it, but it works on my machine (both Windows compiled, and running via Linux WSL). May need more testing.

### CODE REVIEW

File: nvpy/view.py [hunk 1]
```diff
@@ -440,12 +441,24 @@ def __init__(self, master, font_family, font_size, config: NotesListConfig):
...
         f = tkFont.Font(family=font_family, size=font_size)
+
+        # Calculate the text width in characters based on the font
+        # and the desired pixel width of the frame, if the pixel_width
+        # is set at all.
+        text_width = 30
+        try:
+            if config.pixel_width:
+                sentence = 'MmMmMmMmMmMmMmMmMmMmMmMmMmMmMm'
+                text_width = int(float(config.pixel_width)/((float(f.measure(sentence)))*.64)*len(sentence))
```
Comment (mleo2003) [inline nvpy/view.py]: @yuuki0xff I really hate this calculation. I had to add the *.64 on my machine to get the pixel width calculated to be anywhere near what the vale is saved to in the file from one time opening the app to the next. If you know a better way to figure out how wide it should be (or how to make the Frame width be what this follows), I'm all for it. That, or take  out this try/except, and let the default just be what it is on start.

Comment (yuuki0xff) [reply nvpy/view.py]: You can specify width and height in pixels by using Frame. See https://stackoverflow.com/a/14897186 for details.

Changing the initial size of the Frame should work.
https://github.com/cpbotha/nvpy/blob/272f6dfd771975b8a041f8c9e81733a8beead4ec/nvpy/view.py#L1417

**Comment by mleo2003** [comment]
@yuuki0xff Thanks for the pointers, that is smarter for saving. As for the frame trick, I had to add the propagate false statement, as well as +17 on save to keep the frame from shrinking on every save (it always shrunk by 17 on save).

**Comment by yuuki0xff** [comment]
Thank you for contributions.

**Pull request merged**